sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLDialectSpecTest.java

```diff
@@ -3755,6 +3755,8 @@ private void initializeCalciteEnvironmentWithContext(Context... extraContext) {
             .defaultSchema(defaultSchemaPlus)
             .traitDefs(traitDefs)
             .context(Contexts.of(contexts))
+            // TODO[BEAM-8630]: change to BeamRuleSets.getZetaSqlRuleSets()
```

Comment:
We should get this from ZetaSQLQueryPlanner...

Reply:
Done.